src/ResponseCache.php

```diff
@@ -54,6 +55,17 @@ public function cacheResponse(Request $request, Response $response, $lifetimeInS
             ($lifetimeInSeconds) ? intval($lifetimeInSeconds) : $this->cacheProfile->cacheRequestUntil($request)
         );
 
+        foreach (config('responsecache.replacers', []) as $replacerClass) {
```

Comment:
Let's replace this by a collection loop. In the first `map` operation you can can convert the string to an actual class. In the second `each` operation you can just typehint `Replacer`.